src/components/AnonymousChat/child/ChannelFollowButton.tsx

```diff
@@ -42,13 +42,13 @@ const styles = StyleSheet.create({
     paddingVertical: dimen.normalizeDimen(9.5),
     paddingHorizontal: dimen.normalizeDimen(14),
     borderRadius: 8,
-    backgroundColor: colors.darkBlue
+    backgroundColor: COLORS.signed_primary
   },
   followingBtn: {
     paddingVertical: dimen.normalizeDimen(9.5),
     paddingHorizontal: dimen.normalizeDimen(4.5),
     borderWidth: 1,
-    borderColor: colors.darkBlue,
+    borderColor: COLORS.signed_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color and border color for the button have been updated to use the new `COLORS.signed_primary` value. This change is consistent with the PR description and seems to be part of a larger refactoring effort to centralize color management in the application.
```diff
-    backgroundColor: colors.darkBlue
+    backgroundColor: COLORS.signed_primary

-    borderColor: colors.darkBlue,
+    borderColor: COLORS.signed_primary,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->